### ISSUE TITLE
VSCode not auto discovered for file opener preference in Windows 10

### ISSUE THREAD

**Issue by diminutivesloop** [cypress-io/cypress#15080]
### Current behavior

When changing the File Opener Preference from the test runner GUI on my Windows 10 machine I must manually specify the path to the executable for VSCode.

### Desired behavior

VSCode should be auto discovered and listed as an option. I believe this is because Cypress expects the VSCode executable to be named `code.exe` as defined in [`env-editors.ts`](https://github.com/cypress-io/cypress/blob/4d76049d2c419d77968beafcdda45585744fc542/packages/server/lib/util/env-editors.ts#L132). However on my Windows 10 machine the VSCode cli command is installed as `code` with no `.exe` extension. 
```
U:\>where code
C:\Users\npalmer\AppData\Local\Programs\Microsoft VS Code\bin\code
C:\Users\npalmer\AppData\Local\Programs\Microsoft VS Code\bin\code.cmd

U:\>where code.exe
INFO: Could not find files for the given pattern(s).
```
I don't recall having modified this so I assume this is the same for other VSCode users on Windows 10. 

### Versions
Cypress 6.4.0
VSCode 1.53.2 (user setup)
Windows 10 (18363.1256)

**Comment by jennifer-shehane** [comment]
Yes, I can see this on my Windows machine also. I just installed VS Code to the default folder and it is not found by Cypress by default.
